### PR TITLE
feat[specs]: add vercel-cli and signing git commit 

### PR DESCRIPTION
**Pull request by umarhadi**
Hi everyone!

I added 13 workflows for vercel-cli and 1 workflow for git

Thank you 🙌🏻

### CODE REVIEW

**Comment by elviskahoro** [comment]
@umarhadi Are you in our discord, if so what's your username so I can add the Contributor role?

**Comment by umarhadi** [comment]
> @umarhadi Are you in our discord, if so what's your username so I can add the Contributor role?

Yes, I’m in Discord. umar#8211 is my username, Thank you! ✨

**Comment by elviskahoro** [comment]
Added you

**Review by alokedesai — CHANGES_REQUESTED** [review]
Apologies for the delay in reviewing here--I had reviewed this PR months ago but forgot to hit submit 😢 

High level--I think the most useful workflows are ones that involve running a hard-to-remember command with a lot of arguments or flags. 

Workflows that are one-to-one copied from an existing CLI that just add single flags or arguments are generally much less useful since they are easier to remember.

Would you mind combing through some of the these workflows and removing the ones that are 1:1 ports of ports of the documentation of the vercel CLI? Thanks so much!

File: specs/git/commit_message_with_sign.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Commit message with sign
+command: git commit -S -s -m "{{message}}"
+tags:
+  - git
+description: "Using GPG or S/MIME, you can sign tags and commits locally. These tags or commits are marked as verified on GitHub Enterprise Server so other people can be confident that the changes come from a trusted source."
```
[the remark this answers is not in the document]
Comment (umarhadi) [reply specs/git/commit_message_with_sign.yaml]: done :)

**Comment by umarhadi** [comment]
it's still relevant, thanks for the review. have a good day!

**Comment by alokedesai** [comment]
@umarhadi Ping on the high level feedback here around some workflows that just seem to be regurgitations of a `man` page. There still seem to a few of these workflows (such as `specs/vercel/deploy_vercel_projects_from_specific_project_path.yaml` and `specs/vercel/perform_a_binary_search_upon_a_set_of_deployments_in_a_vercel_project.yaml`). LMK if you are still actively working on these--if not we should close out. Thanks!

**Pull request closed without merging**